fem/conduitdatacollection.cpp

```diff
@@ -0,0 +1,1189 @@
+   // name containing "_attribute", that doesn't contain "boundary"
+   std::string main_att_name = "";
+
+   const Node &n_fields = n_mesh["fields"];
+   NodeConstIterator itr = n_fields.children();
+
+   while ( itr.has_next() && main_att_name == "" )
+   {
+      itr.next();
+      std::string fld_name = itr.name();
+      if ( fld_name.find("boundary")   == std::string::npos &&
+           fld_name.find("_attribute") != std::string::npos )
+      {
+         main_att_name = fld_name;
+      }
+   }
+
+   if ( main_att_name != "" )
+   {
+      const Node &n_mesh_atts_vals = n_fields[main_att_name]["values"];
+
+      // mfem requires ints, we could have int64s, etc convert if necessary
+      if (n_mesh_atts_vals.dtype().is_int() &&
+          n_mesh_atts_vals.is_compact() )
+      {
+         mesh_atts = n_mesh_atts_vals.value();
+      }
+      else
+      {
+         Node &n_mesh_atts_vals_conv = n_conv["fields"][main_att_name]["values"];
+         n_mesh_atts_vals.to_int_array(n_mesh_atts_vals_conv);
+         mesh_atts = n_mesh_atts_vals_conv.value();
+      }
+
+      num_mesh_atts_entires = n_mesh_atts_vals.dtype().number_of_elements();
+   }
+   else
+   {
+      // Skipping Mesh Attribute Data
+   }
+
+   // for the boundary attributes check for first occurrence of field with
+   // name containing "_attribute", that also contains "boundary"
+   std::string bnd_att_name = "";
+   itr = n_fields.children();
+
+   while ( itr.has_next() && bnd_att_name == "" )
+   {
+      itr.next();
+      std::string fld_name = itr.name();
+      if ( fld_name.find("boundary")   != std::string::npos &&
+           fld_name.find("_attribute") != std::string::npos )
+      {
+         bnd_att_name = fld_name;
+      }
+   }
+
+   if ( bnd_att_name != "" )
+   {
+      // Info: "Getting Boundary Attribute Data"
+      const Node &n_bndry_atts_vals =n_fields[bnd_att_name]["values"];
+
+      // mfem requires ints, we could have int64s, etc convert if necessary
+      if ( n_bndry_atts_vals.dtype().is_int() &&
+           n_bndry_atts_vals.is_compact())
+      {
+         bndry_atts = n_bndry_atts_vals.value();
+      }
+      else
+      {
+         Node &n_bndry_atts_vals_conv = n_conv["fields"][bnd_att_name]["values"];
+         n_bndry_atts_vals.to_int_array(n_bndry_atts_vals_conv);
+         bndry_atts = n_bndry_atts_vals_conv.value();
+      }
+
+      num_bndry_atts_entires = n_bndry_atts_vals.dtype().number_of_elements();
+
+   }
+   else
+   {
+      // Skipping Boundary Attribute Data
+   }
+
+   // Info: "Number of Vertices: " << num_verts  << endl
+   //         << "Number of Mesh Elements: "    << num_mesh_ele   << endl
+   //         << "Number of Boundary Elements: " << num_bndry_ele  << endl
+   //         << "Number of Mesh Attribute Entries: "
+   //         << num_mesh_atts_entires << endl
+   //         << "Number of Boundary Attribute Entries: "
+   //         << num_bndry_atts_entires << endl);
```

Comment:
@cyrush, do you want to keep this for debugging purposes? I'm asking because I get warnings:
```ascii
fem/conduitdatacollection.cpp: In static member function ‘static mfem::Mesh* mfem::ConduitDataCollection::BlueprintMeshToMesh(const conduit::Node&, bool)’:
fem/conduitdatacollection.cpp:326:8: warning: variable ‘num_mesh_atts_entires’ set but not used [-Wunused-but-set-variable]
    int num_mesh_atts_entires = 0;
        ^
fem/conduitdatacollection.cpp:327:8: warning: variable ‘num_bndry_atts_entires’ set but not used [-Wunused-but-set-variable]
    int num_bndry_atts_entires = 0;
        ^
```
that I want to fix. If you want to keep the comment, I'll just comment out the two variables. Otherwise, I'll remove the comment and the variables.

Reply:
Thats a left over from an older sanity check. If you are ok with it, lets keep the comment and comment out the vars just in case the context is helpful in the future.

Reply:
OK, I'll comment out the variables. Thanks.